docs/scripts/populatePickersDemos.js

```diff
@@ -0,0 +1,77 @@
+const { run: jscodeshift } = require('jscodeshift/src/Runner');
+/**
+ * Transpiles TypeScript demos to formatted JavaScript.
+ * Can be used to verify that JS and TS demos are equivalent. No introduced change
+ * would indicate equivalence.
```

Comment:
This description is probably not the most correct, is it? 🤔 
This script would only apply/change `components/component` prop on existing demos files.